qsirecon/cli/parser.py

```diff
@@ -612,13 +296,6 @@ def _bids_filter(value, parser):
         type=Path,
         help="Directory containing freesurfer outputs to be integrated into recon.",
     )
-    g_recon.add_argument(
-        "--skip-odf-reports",
-        "--skip_odf_reports",
-        action="store_true",
-        default=False,
-        help="run only reconstruction, assumes preprocessing has already completed.",
-    )
```

Comment:
The help for this is incorrect - it should say that this skips 3d-rendered images, which often cause crashes when run on machines without monitor

Reply:
And it should be kept. Lots of HPCs struggle to make the odf reports